composer/packages/doc-search/src/components/list.tsx

```diff
@@ -0,0 +1,332 @@
+                // change search term to lowercase
+                const filter = searchTxt.toLowerCase();
+                // check to see if the current list item includes the search term
+                // If it does, it will be added to newList. Using lowercase eliminates
+                // issues with capitalization in search terms and search content
+                return lc.includes(filter);
+            });
+
+            newFunctionsList = currentFunctionsList.filter((item: any) => {
+                const lc = item.id.toLowerCase();
+                const filter = searchTxt.toLowerCase();
+                return lc.includes(filter);
+            });
+
+            newObjectsList = currentObjectsList.filter((item: any) => {
+                const lc = item.id.toLowerCase();
+                const filter = searchTxt.toLowerCase();
+                return lc.includes(filter);
+            });
+
+            newRecordsList = currentRecordsList.filter((item: any) => {
+                const lc = item.id.toLowerCase();
+                const filter = searchTxt.toLowerCase();
+                return lc.includes(filter);
+            });
+
+            newConstantsList = currentConstantsList.filter((item: any) => {
+                const lc = item.id.toLowerCase();
+                const filter = searchTxt.toLowerCase();
+                return lc.includes(filter);
+            });
+
+            newErrorsList = currentErrorsList.filter((item: any) => {
+                const lc = item.id.toLowerCase();
+                const filter = searchTxt.toLowerCase();
+                return lc.includes(filter);
+            });
+
+            newTypesList = currentTypesList.filter((item: any) => {
+                const lc = item.id.toLowerCase();
+                const filter = searchTxt.toLowerCase();
+                return lc.includes(filter);
+            });
+
+        } else {
+            if (mainDiv != null) {
+                mainDiv.classList.remove('hidden');
+            }
+        }
+        // Set the filtered state based on what our rules added to newList
+        this.setState({
+            filteredModules: newModuleList,
+            filteredFunctions: newFunctionsList,
+            filteredObjects: newObjectsList,
+            filteredRecords: newRecordsList,
+            filteredConstants: newConstantsList,
+            filteredErrors: newErrorsList,
+            filteredTypes: newTypesList
+        });
+    }
+
+    render() {
+        return (
+            <div>
+                <div className="ui fluid icon input search-bar">
+                    <input type="text" id="searchBox" onKeyUp={this.handleChange} placeholder="Search..." />
+                    <i className="search icon"></i>
+                </div>
+
+                {/* <input type="text" id="searchBox" className="input" onKeyUp={this.handleChange} placeholder="Search..." defaultValue={this.state.searchText} /> */}
```

Comment:
Shall we remove these commented code lines?

Reply:
Added changes